src/Facade/Services/PartsStorageTypeFacadeService.cs

```diff
@@ -51,14 +59,16 @@ protected override async Task<PartsStorageType> CreateFromResourceAsync(
                 throw new PartsStorageTypeException("Part Storage Type Already Exists");
             }
 
+            var bridgeId = this.databaseService.GetIdSequence("PARTS_STORAGE_TYPES_ID_SEQ");
```

Comment:
I've been favouring inserting the id in db triggers these days, but this works too